DapperLearning/Models/User.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace DapperLearning
+{
+    internal class User
+    {
+        public User()
+        {
+            Posts = new List<Post>();
+        }
+
+        public Guid Id { get; set; }
+        public String FirstName { get; set; }
```

Comment:
use `string` not `String`. 
There is not real reason why, but it's common practice to use like this
https://www.c-sharpcorner.com/blogs/c-sharp-string-vs-string-the-difference